lib/rules/array-bracket-newline.js

```diff
@@ -15,8 +15,26 @@ const astUtils = require("./utils/ast-utils");
 /** @type {import('../shared/types').Rule} */
 module.exports = {
     meta: {
-        deprecated: true,
-        replacedBy: [],
+        deprecated: {
+            message: "Stylistic rules are being moved out of ESLint core.",
```

Comment:
```suggestion
            message: "Formatting rules are being moved out of ESLint core.",
```

We'll still have stylistic but not formatting rules in ESLint core.

(the same suggestion for all other rules with this message)